check-dependent-jobs/action.yml

```diff
@@ -0,0 +1,28 @@
+name: check-dependent-jobs
+description: |
+  Evaluates the combined the status results of the provided needs context.
+inputs:
+  needs:
```

Comment:
wonder if `jobs` input is more meaningful

Reply:
https://github.com/elastic/oblt-actions/pull/16/commits/e14f866c4f08a0d77f9463377053cb7c8d79c2a3